js/main.js

```diff
@@ -666,7 +666,12 @@ const MM = (function () {
 			}
 
 			// Further implementation is done in the private method.
-			updateDom(module, updateOptions);
+			let update_promise = updateDom(module, updateOptions);
```

Comment:
why do we need the update_promise variable? 
why not write updateDom(...).then()... ?

Reply:
@rejas 

No specific reason, I was originally doing some other things with the promise during my testing to find the bug. I'll get that updated in a new commit now.

Reply:
@rejas updated